neoscore/core/path.py

```diff
@@ -62,6 +62,10 @@ def __init__(
         self._z_index = z_index
         self._rotation = rotation
         self.elements: list[PathElement] = []
+        # I assumed that this needed declaring at the start in order for a close_sub_path to close
+        # even on a minor Path command. Especially as the docs state that close_path should
+        # draw a line back to ORIGIN. But this goes against the issue ticket.
+        self._current_subpath_start: tuple[Point, parent] = (pos, self)
```

Comment:
Maybe initialize it to `None`, then have the type be `Optional[tuple[...]]`, then `close_subpath` can check if it's `None` and raise an error if it is? This way we error if `close_subpath` is called when no path exists, which is reasonable behavior IMO.